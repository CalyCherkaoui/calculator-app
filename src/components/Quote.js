import React from 'react';

const Quote = () => (
  <div className="quote">
    <h1 className="quote_title">Daily dose of wisdom</h1>
    <p className="quote_text">
      Mathematics is the most beautiful and most powerful creation of the human spirit.
      <span>-by Stefan Banach</span>
    </p>

  </div>
);

export default Quote;
