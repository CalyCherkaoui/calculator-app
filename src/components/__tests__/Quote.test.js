import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

describe(
  'Quote page',
  () => {
    it('Matches Quote page snapshot', () => {
      const comp = renderer.create(<Quote />);
      expect(comp).toMatchSnapshot();
    });
  },
);
