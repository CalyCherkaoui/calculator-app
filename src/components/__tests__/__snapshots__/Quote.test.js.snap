// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page Matches Quote page snapshot 1`] = `
<div
  className="quote"
>
  <h1
    className="quote_title"
  >
    Daily dose of wisdom
  </h1>
  <p
    className="quote_text"
  >
    Mathematics is the most beautiful and most powerful creation of the human spirit.
    <span>
      -by Stefan Banach
    </span>
  </p>
</div>
`;
